data_pipeline/config.py

```diff
@@ -304,7 +304,7 @@ def key_location(self):
         """Directory in which to look for key to encrypt pii."""
         return data_pipeline_conf.read_string(
             'key_location',
-            default='/nail/srv/configs/data_pipeline/'
+            default='./'
```

Comment:
Can we add a comment/warning here that the default key is a test key and shouldn't be used in production? (Maybe also rename the keyfile if possible)